locales/fr.json

```diff
@@ -122,6 +122,7 @@
     "mail_alias_remove_failed": "Impossible de supprimer l'alias courriel « {mail:s} »",
     "mail_domain_unknown": "Le domaine « {domain:s} » du courriel est inconnu",
     "mail_forward_remove_failed": "Impossible de supprimer le courriel de transfert « {mail:s} »",
+    "mail_unavailable": "Adresse mail indisponible",
```

Comment:
(We are not supposed to edit fr.json, this usually breaks weblate :s)

Reply:
well... it's because weblate is already broken that I did that. But ok. Will be removed. Only english will stay.

Reply:
Done